src/DataTable/tests/BulkActions.test.jsx

```diff
@@ -158,40 +122,30 @@ describe('<BulkActions />', () => {
       const button = wrapper.find(Button).at(1);
       button.simulate('click');
       expect(onClickSpy).toHaveBeenCalledTimes(1);
-      expect(onClickSpy).toHaveBeenCalledWith({
-        isEntireTableSelected: true,
-        selectedRows: selectedFlatRows,
-      });
     });
   });
   describe('two actions on click', () => {
     it('performs the primary button action on click', () => {
       const onClickSpy = jest.fn();
       const wrapper = mount(
         <BulkActionsWrapper
-          value={{ ...instance, bulkActions: [{ ...firstAction, handleClick: onClickSpy }, secondAction] }}
+          value={{ ...instance, bulkActions: [<FirstAction onClick={onClickSpy} />, <SecondAction />] }}
         />,
       );
       const button = wrapper.find(Button).at(1);
       button.simulate('click');
       expect(onClickSpy).toHaveBeenCalledTimes(1);
-      expect(onClickSpy).toHaveBeenCalledWith({
-        selectedRows: selectedFlatRows,
-      });
     });
     it('performs the second button action on click', () => {
       const onClickSpy = jest.fn();
       const wrapper = mount(
         <BulkActionsWrapper
-          value={{ ...instance, bulkActions: [firstAction, { ...secondAction, handleClick: onClickSpy }] }}
+          value={{ ...instance, bulkActions: [<FirstAction />, <FirstAction onClick={onClickSpy} />] }}
```

Comment:
nit: should this be `[<FirstAction />, <SecondAction onClick={onClickSpy} />]`?

Reply:
👍